src/test/checkqueue_tests.cpp

```diff
@@ -9,7 +9,7 @@
 #include <util/time.h>
 
 #include <boost/test/unit_test.hpp>
-#include <boost/thread.hpp>
+#include <boost/thread/thread.hpp>
```

Comment:
1c7fdb71b32bd91f6ae10313169d9eec63d46e3e: This commit should probably go last, because some of the changes are removed earlier anyway

Reply:
In that case in the commit _"bench: Use CCheckQueue local thread pool"_ the `lint-includes.sh` linter will produce:
```
Good job! The Boost dependency "boost/thread/thread.hpp" is no longer used.
Please remove it from EXPECTED_BOOST_INCLUDES in ./test/lint/lint-includes.sh
to make sure this dependency is not accidentally reintroduced.
```
which seems a bit falsely.

Reply:
I hope that no one is running the linters on every commit. They are meant to warn authors and reviewers as well as maintainers of code smell in a pull request as a whole. They are not meant as unit tests that absolutely must pass on every commit.

Reply:
[Updated](https://github.com/bitcoin/bitcoin/pull/18710#issuecomment-619008172).